groovy/gradle/src/org/netbeans/modules/gradle/execute/EscapeProcessingOutputStream.java

```diff
@@ -29,9 +29,10 @@
  * @author Laszlo Kishalmi
  */
 class EscapeProcessingOutputStream extends OutputStream {
-    
+
     boolean esc;
     boolean csi;
+    boolean closed = false;
```

Comment:
I wonder if some kind of synchronization is needed for the field? (Maybe AtomicBoolean instead of plain boolean?)